### PR TITLE
Add student label to evaluation interface

### LINKED ISSUE #2676: Show student labels in evaluation interface
Is it possible to add the label of the student next to his/her name? Some students who don't have a special status (faciliteiten, bijzonder statuut) sometimes submit solutions after the deadline, which isn't allowed. Having the label next to the students name, would allow the teacher to monitor if the late submission is permitted. Another advantage is that the label reminds the teacher to choose the latest submission for students with extra time.

image

### PR DESCRIPTION
**Pull request by TimonDB**
This pull request shows the student labels in the evaluation overview. If more than 1 label is present, the next labels are shown beneath the name.

Screenshot from 2021-08-30 10-24-52 Screenshot from 2021-08-30 10-24-33

Closes #2676.

### CODE REVIEW

**Comment by TimonDB** [comment]
Someone who knows why the system test keeps failing?

**Comment by bmesuere** [comment]
```
      ActionView::Template::Error: undefined local variable or method `users_labels' for #<ActionView::Base:0x00000000028410>
      Did you mean?  users_url
          app/views/feedbacks/_feedback_actions.html.erb:111
          app/views/feedbacks/show.js.erb:1
          app/controllers/scores_controller.rb:40:in `block (2 levels) in destroy'
          app/controllers/scores_controller.rb:39:in `destroy'
      # app/views/feedbacks/_feedback_actions.html.erb:111
      # app/views/feedbacks/show.js.erb:1
      # app/controllers/scores_controller.rb:40
      # app/controllers/scores_controller.rb:39
      # app/controllers/application_controller.rb:170
```

**Review by chvp — CHANGES_REQUESTED** [review]
Not a huge fan of the stacked labels. What's the issue with showing them one after another horizontally?

I'm also pretty sure we use an orange hue for the labels elsewhere, so I would use that same color here.

**Comment by TimonDB** [comment]
> Not a huge fan of the stacked labels. What's the issue with showing them one after another horizontally?

Showing them horizontally can cause the student's name to be truncated quickly as shown in the screenshot... Maybe it can be checked how many labels there are; (1) in case of one label, it is shown next to the name. (2) if more than 1 label, it is shown beneath the name? Or we can always show them beneath the name
Screenshot from 2021-08-24 10-17-33

**Comment by bmesuere** [comment]
I'll submit another pull request today which reworks the label style. Maybe put this one on hold until that lands.

**Comment by bmesuere** [comment]
You can rebase this on develop which now includes the redesigned labels.

File: app/controllers/feedbacks_controller.rb [hunk 1]
```diff
@@ -23,13 +23,14 @@ def show
...
                              .series
                              .course
                              .course_memberships
-                             .includes(:course_labels, :user)
-                             .map { |m| [m.user, m.course_labels] }
-                             .to_h
+                             .find_by(user_id: @feedback.user)
+                             .course_membership_labels
+                             .includes(:course_label)
+                             .map { |m| m.course_label }
```
Comment (chvp) [inline app/controllers/feedbacks_controller.rb]:
```suggestion
                             .course_labels
```

**Review by chvp — APPROVED** [review]

**Pull request merged**